xds/src/main/java/io/grpc/xds/XdsClientImpl.java

```diff
@@ -786,25 +804,34 @@ private void handleRdsResponse(DiscoveryResponse rdsResponse) {
 
     // Notify the ConfigWatcher if this RDS response contains the most recently requested
     // RDS resource.
-    if (clusterName != null) {
+    if (routes != null) {
       if (rdsRespTimer != null) {
         rdsRespTimer.cancel();
         rdsRespTimer = null;
       }
-      logger.log(XdsLogLevel.INFO, "Found cluster name: {0}", clusterName);
-      ConfigUpdate configUpdate = ConfigUpdate.newBuilder().setClusterName(clusterName).build();
+
+      // Found clusterName in the in-lined RouteConfiguration.
+      String clusterName = routes.get(routes.size() - 1).routeAction.get().cluster;
+      if (!enablePathMatching) {
+        logger.log(XdsLogLevel.INFO, "Found cluster name: {0}", clusterName);
+      } else {
+        logger.log(
+            XdsLogLevel.INFO,
+            "Found routes: {0}", routes);
```

Comment:
This INFO log will print a list of complex structures, which is not appropriate for INFO level logs. Use INFO level logging to print concise descriptions for those routes or just print out the number of routes found. INFO level log messages are intended to be concise and informative.

Reply:
Changed to DEBUG level. INFO level only logs count now.